xcm/xcm-executor/src/lib.rs

```diff
@@ -41,6 +41,9 @@ pub use config::Config;
 /// The XCM executor.
 pub struct XcmExecutor<Config>(PhantomData<Config>);
 
+/// The maximum recursion limit for `execute_xcm` and `execute_effects`.
+pub const MAX_RECURSION_LIMIT: usize = 16;
```

Comment:
should probably a configuration of `trait ExecuteXcm`

Reply:
Talked with @shawntabrizi about it, we decided that we want every XCM to have the same limit, instead of it being configurable by whoever implements the `ExecuteXcm` trait.